openhtf/io/user_input.py

```diff
@@ -31,17 +31,20 @@
 import threading
 import uuid
 
-import gflags
+from openhtf.util import argv
+
 
 if platform.system() != 'Windows':
   import termios
 
 _LOG = logging.getLogger(__name__)
 
-FLAGS = gflags.FLAGS
-gflags.DEFINE_integer('prompt_timeout_s',
-                       None,
-                      'User prompt timeout in seconds.')
+ARG_PARSER = argv.ModuleParser()
+ARG_PARSER.add_argument(
+    '--prompt_timeout_s', type=int, action=argv.StoreInModule,
+    target='openhtf.io.user_input.DEFAULT_TIMEOUT_S',
```

Comment:
hm, i think it might be better to do `'.'.join((__module__, 'DEFAULT_TIMEOUT_S'))` that way you don't have a magic string in the file that depends on the file location.